cypress/integration/previewApi.spec.js

```diff
@@ -41,12 +47,4 @@ describe("Preview an API", () => {
                 }
             });
     });
-
-    it("When the user clicks on an API name they are directed to the SwaggerHub page", function() {
-        cy.get(".apiPreview").find("a").first().click();
-        const apiUrl = this.apiData.properties.filter( property => property.type === "Swagger")[0].url;
-        const expectedUrl = apiUrl.replace("api", "app");
-        cy.url().should('include', expectedUrl);
-    });
-
```

Comment:
We need to have failing test scenarios so can make sure those are handled as expected